### ISSUE TITLE
cannot start any container: error: Failed to run: /usr/bin/lxd forkstart mycontainer /var/lib/lxd/containers /var/log/lxd/mycontainer/lxc.conf: 

### ISSUE THREAD

**Issue by gaetanquentin** [lxc/lxd#3090]
# Required information

 * Distribution: ubuntu
 * Distribution version: 16.04
 * The output of "lxc info": 
config:
  core.https_address: :8443
  core.trust_password: true
api_extensions:
- storage_zfs_remove_snapshots
- container_host_shutdown_timeout
- container_syscall_filtering
- auth_pki
- container_last_used_at
- etag
- patch
- usb_devices
- https_allowed_credentials
- image_compression_algorithm
- directory_manipulation
- container_cpu_time
- storage_zfs_use_refquota
- storage_lvm_mount_options
- network
- profile_usedby
- container_push
- container_exec_recording
- certificate_update
- container_exec_signal_handling
- gpu_devices
- container_image_properties
- migration_progress
- id_map
- network_firewall_filtering
- network_routes
- storage
- file_delete
- file_append
- network_dhcp_expiry
- storage_lvm_vg_rename
- storage_lvm_thinpool_rename
- network_vlan
- image_create_aliases
api_status: stable
api_version: "1.0"
auth: trusted
public: false
environment:
  addresses:
  - :8443
  architectures:
  - x86_64
  - i686
  certificate: |
    -----BEGIN CERTIFICATE-----
     -----END CERTIFICATE-----
  certificate_fingerprint: ea854278d6e269bb441c2b59b3407ff2a90459486478cefc439d9750e3b2afd9
  driver: lxc
  driver_version: 2.0.7
  kernel: Linux
  kernel_architecture: x86_64
  kernel_version: 4.4.0-66-generic
  server: lxd
  server_pid: 5988
  server_version: "2.11"
  storage: btrfs
  storage_version: "4.4"

# Issue description

Don't know what i have done bad, but i cannot start containers anymore:
lxc start mycontainer:

error: Failed to run: /usr/bin/lxd forkstart mycontainer /var/lib/lxd/containers /var/log/lxd/mycontainer/lxc.conf: 

# Steps to reproduce

 1. lxc start mycontainer
 

# Information to attach
debug output:

DBUG[03-19|09:32:05] handling                                 ip=@ method=GET url=/1.0
DBUG[03-19|09:32:05] Initializing a BTRFS driver. 
DBUG[03-19|09:32:05] 
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"config": {
				"core.https_address": ":8443",
				"core.trust_password": true
			},
			"api_extensions": [
				"storage_zfs_remove_snapshots",
				"container_host_shutdown_timeout",
				"container_syscall_filtering",
				"auth_pki",
				"container_last_used_at",
				"etag",
				"patch",
				"usb_devices",
				"https_allowed_credentials",
				"image_compression_algorithm",
				"directory_manipulation",
				"container_cpu_time",
				"storage_zfs_use_refquota",
				"storage_lvm_mount_options",
				"network",
				"profile_usedby",
				"container_push",
				"container_exec_recording",
				"certificate_update",
				"container_exec_signal_handling",
				"gpu_devices",
				"container_image_properties",
				"migration_progress",
				"id_map",
				"network_firewall_filtering",
				"network_routes",
				"storage",
				"file_delete",
				"file_append",
				"network_dhcp_expiry",
				"storage_lvm_vg_rename",
				"storage_lvm_thinpool_rename",
				"network_vlan",
				"image_create_aliases"
			],
			"api_status": "stable",
			"api_version": "1.0",
			"auth": "trusted",
			"public": false,
			"environment": {
				"addresses": [
					":8443"
				],
				"architectures": [
					"x86_64",
					"i686"
				],
				"certificate": "-XXX--END CERTIFICATE-----\n",
				"certificate_fingerprint": "",
				"driver": "lxc",
				"driver_version": "2.0.7",
				"kernel": "Linux",
				"kernel_architecture": "x86_64",
				"kernel_version": "4.4.0-66-generic",
				"server": "lxd",
				"server_pid": 2681,
				"server_version": "2.11",
				"storage": "btrfs",
				"storage_version": "4.4"
			}
		}
	} 
DBUG[03-19|09:32:05] handling                                 ip=@ method=GET url=/1.0/containers/mycontainer
DBUG[03-19|09:32:05] 
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"architecture": "x86_64",
			"config": {
				"volatile.base_image": "ea4426a50682e2170c10309227b6b7ed8cb530ba64ccd28bc986b7088cc5d820",
				"volatile.eth0.hwaddr": "00:16:3e:8d:68:e0",
				"volatile.eth1.hwaddr": "00:16:3e:fd:02:a5",
				"volatile.eth2.hwaddr": "00:16:3e:f5:fd:42",
				"volatile.last_state.idmap": "[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":362144,\"Nsid\":0,\"Maprange\":65536},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":362144,\"Nsid\":0,\"Maprange\":65536}]",
				"volatile.last_state.power": "RUNNING"
			},
			"devices": {
				"root": {
					"path": "/",
					"pool": "default",
					"type": "disk"
				}
			},
			"ephemeral": false,
			"profiles": [
				"front"
			],
			"created_at": "2016-11-10T01:25:00Z",
			"expanded_config": {
				"volatile.base_image": "ea4426a50682e2170c10309227b6b7ed8cb530ba64ccd28bc986b7088cc5d820",
				"volatile.eth0.hwaddr": "00:16:3e:8d:68:e0",
				"volatile.eth1.hwaddr": "00:16:3e:fd:02:a5",
				"volatile.eth2.hwaddr": "00:16:3e:f5:fd:42",
				"volatile.last_state.idmap": "[{\"Isuid\":true,\"Isgid\":false,\"Hostid\":362144,\"Nsid\":0,\"Maprange\":65536},{\"Isuid\":false,\"Isgid\":true,\"Hostid\":362144,\"Nsid\":0,\"Maprange\":65536}]",
				"volatile.last_state.power": "RUNNING"
			},
			"expanded_devices": {
				"eth0": {
					"name": "eth0",
					"nictype": "bridged",
					"parent": "lxdbr0",
					"type": "nic"
				},
				"eth1": {
					"name": "eth1",
					"nictype": "bridged",
					"parent": "lxdbr1",
					"type": "nic"
				},
				"eth2": {
					"name": "eth2",
					"nictype": "bridged",
					"parent": "lxdbr2",
					"type": "nic"
				},
				"root": {
					"path": "/",
					"pool": "default",
					"type": "disk"
				}
			},
			"name": "mycontainer",
			"stateful": false,
			"status": "Stopped",
			"status_code": 102,
			"last_used_at": "2017-03-19T08:31:21.749315618Z"
		}
	} 
DBUG[03-19|09:32:05] handling                                 ip=@ method=PUT url=/1.0/containers/mycontainer/state
DBUG[03-19|09:32:05] 
	{
		"action": "start",
		"force": false,
		"stateful": false,
		"timeout": 0
	} 
DBUG[03-19|09:32:05] New task operation: ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7 
DBUG[03-19|09:32:05] Started task operation: ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7 
DBUG[03-19|09:32:05] 
	{
		"type": "async",
		"status": "Operation created",
		"status_code": 100,
		"operation": "/1.0/operations/ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7",
		"error_code": 0,
		"error": "",
		"metadata": {
			"id": "ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7",
			"class": "task",
			"created_at": "2017-03-19T09:32:05.194592662+01:00",
			"updated_at": "2017-03-19T09:32:05.194592662+01:00",
			"status": "Running",
			"status_code": 103,
			"resources": {
				"containers": [
					"/1.0/containers/mycontainer"
				]
			},
			"metadata": null,
			"may_cancel": false,
			"err": ""
		}
	} 
DBUG[03-19|09:32:05] handling                                 ip=@ method=GET url=/1.0/operations/ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7/wait
DBUG[03-19|09:32:05] Initializing a BTRFS driver. 
DBUG[03-19|09:32:05] Mounting BTRFS storage volume for container "mycontainer" on storage pool "default". 
DBUG[03-19|09:32:05] Mounting BTRFS storage pool "default". 
DBUG[03-19|09:32:05] Mounted BTRFS storage volume for container "mycontainer" on storage pool "default". 
DBUG[03-19|09:32:05] Mounting BTRFS storage volume for container "mycontainer" on storage pool "default". 
DBUG[03-19|09:32:05] Mounting BTRFS storage pool "default". 
DBUG[03-19|09:32:05] Mounted BTRFS storage volume for container "mycontainer" on storage pool "default". 
INFO[03-19|09:32:05] Starting container                       action=start created=2016-11-10T01:25:00+0000 ephemeral=false name=mycontainer stateful=false used=2017-03-19T08:31:21+0000
DBUG[03-19|09:32:05] handling                                 ip=@ method=GET url=/1.0
DBUG[03-19|09:32:05] Initializing a BTRFS driver. 
DBUG[03-19|09:32:05] 
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"config": {
				"core.https_address": ":8443",
				"core.trust_password": true
			},
			"api_extensions": [
				"storage_zfs_remove_snapshots",
				"container_host_shutdown_timeout",
				"container_syscall_filtering",
				"auth_pki",
				"container_last_used_at",
				"etag",
				"patch",
				"usb_devices",
				"https_allowed_credentials",
				"image_compression_algorithm",
				"directory_manipulation",
				"container_cpu_time",
				"storage_zfs_use_refquota",
				"storage_lvm_mount_options",
				"network",
				"profile_usedby",
				"container_push",
				"container_exec_recording",
				"certificate_update",
				"container_exec_signal_handling",
				"gpu_devices",
				"container_image_properties",
				"migration_progress",
				"id_map",
				"network_firewall_filtering",
				"network_routes",
				"storage",
				"file_delete",
				"file_append",
				"network_dhcp_expiry",
				"storage_lvm_vg_rename",
				"storage_lvm_thinpool_rename",
				"network_vlan",
				"image_create_aliases"
			],
			"api_status": "stable",
			"api_version": "1.0",
			"auth": "trusted",
			"public": false,
			"environment": {
				"addresses": [
					":8443"
				],
				"architectures": [
					"x86_64",
					"i686"
				],
				"certificate": "-----BEGIN CERTIFICATE-----\nMIIGgjCCBGqgAwIBAgIRAP900K6jIf1qTWYwDrq1lYYwDQYJKoZIhvcNAQELBQAw\nPTEcMBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzEdMBsGA1UEAwwUcm9vdEBn\ncXVlbnRpbi1zc2QtdDEwHhcNMTYxMDE4MDIzOTM1WhcNMjYxMDE2MDIzOTM1WjA9\nMRwwGgYDVQQKExNsaW51eGNvbnRhaW5lcnMub3JnMR0wGwYDVQQDDBRyb290QGdx\ndWVudGluLXNzZC10MTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBAJb8\nHa2I1Fo1sWKZwXon4OpK/ckDm7HwRtH18+i1yuiddVt/8rhtGkeAMAJhY4rxZb1w\nJoNu2G42JWLQGhtbDOmnbfvNj9DsB+vhpROPJK90K8SFkN4r9kzfW3UIa9LfOg9S\nM5frjdbCEFnSV7rUTm6abMHf2LC6iiVkbsl0psg2Ke7ENlLYS0g8zn7CeNSnYCzC\nYwTj6t0k4uvLAG1YocYVoY1A3z9w2mp2oZEnXgLByB7MmO2JtchXuvH1TDbq5dAH\nH73OOFZxVo5OyBIua+2h3ZPI2bLLqwFehAkKbD8Fd048vwQpBNGARdvcP+nbFqBy\nu7d7w+W5pcR+VvOxC0mphHNIOZ8NeFc0PoIJnNTkF5Nx7i/3omeO7VT53rRc9T6V\n0QcDtM2moXSjwgut7kM/FI3Qh6X2pkRy9TwSOnjOUTdozBxoBGV50tZci1h1BPsB\n1xN9S1PSlDwAUA8YgZN8a8d//xkpsySrLeaTJ8xMlI8GYSt+IEdFuukLMaoYFUhk\nfBpkBRePV611UU2mMW0/bso6bxRSf/dvuEs1gxmMHdKmC9d890OqNtYuwhX2loDV\nodX6KTMYmatDDzlR33xQ29IeXSfsyRlMutSY27MqEgt/u9Xhjj4w8JqysZaRRZYb\n+oi+nCVedFbYRlVpa45eaTzAvzrujB+d9bGNMyBJAgMBAAGjggF7MIIBdzAOBgNV\nHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDAYDVR0TAQH/BAIwADCC\nAUAGA1UdEQSCATcwggEzgg9ncXVlbnRpbi1zc2QtdDGCDjE3Mi4xNi4wLjIwLzEy\nghxmZTgwOjo3NGMyOjVlYjk6M2RmZDo3NGU0LzY0ghxmZTgwOjpiYzAyOjRhZmY6\nZmUwZTo0Y2ViLzY0ghxmZTgwOjpiYzAyOjRhZmY6ZmUwZTo0Y2ViLzY0ghxmZTgw\nOjpiYzAyOjRhZmY6ZmUwZTo0Y2ViLzY0gg4xNzIuMTYuMTEuMS8yNIIcZmU4MDo6\nYmMwMjo0YWZmOmZlMGU6NGNlYi82NIIOMTcyLjE2LjEyLjEvMjSCHGZlODA6OmJj\nMDI6NGFmZjpmZTBlOjRjZWIvNjSCDjE3Mi4xNi4xMC4xLzI0ghxmZTgwOjpmMDkw\nOjM4ZmY6ZmUzNTo3N2RhLzY0gg4xOTIuMTY4LjAuMS8yMDANBgkqhkiG9w0BAQsF\nAAOCAgEAC43ulECKjZxCAcqjfRkHfC/8CTZbp6fLwEsZe/jyWvsYGwoapm6v7qsl\nb8yrwB0+7VqDRHH2aIGG5OJyB2o6FuPVMZnfV+2XEhmeKbP0uybupT/cjy9HLiRa\nKXuWWMDSKuDnwHQx6iw0KiILrZdM2ooc4tw7x+Ed6GUOnQczXrnxXCr7O9WI4cKt\nY1LeY+dv3MPkdrB8dmLf/olPLkJC9hhaYGOWTIQfU2Y96AQIB64/y8duMGcXcStS\n3x8izZx/agDVC3fYr4DrT7+/tqDr1F06SsuP8CZbwDbO51yJob9gJb7SziB/eQqK\n+Kn2d39c+3ohNyTEO1v7wwYL5NbPS3jaKtt2UBiXjVNPCMJoZlJgA6pN8ZcGy5sX\nCcFq9OM0T2+hpq3AIIojodmxTcOGfDcX1IA+rq6pwyCMJahXUwk84FGEmW9LYR6T\nKI5AToUB6MW98ccfmJtdBWzZf9YDmsWfkjxzjKhc3n89wFhBjZPDEHl9772lmICR\nfFeQTezhVwcS5gH50lH162ffprYCICtI3Xwh0a0y9KFXyspabQ5bRzozBHpjRkf0\nHnLsXvkWauRuQtska9kXkljUVl/ZF+h1kO2PRxOor1+VnxBm1XWBEF+0BoRaVQe2\nfgWcrzbDBCtCAY2TVQ18o6vPi+hsK3WjMRHzvKhd3iWGXb4w/hA=\n-----END CERTIFICATE-----\n",
				"certificate_fingerprint": "",
				"driver": "lxc",
				"driver_version": "2.0.7",
				"kernel": "Linux",
				"kernel_architecture": "x86_64",
				"kernel_version": "4.4.0-66-generic",
				"server": "lxd",
				"server_pid": 2681,
				"server_version": "2.11",
				"storage": "btrfs",
				"storage_version": "4.4"
			}
		}
	} 
DBUG[03-19|09:32:05] handling                                 ip=@ method=GET url=/internal/containers/14/onstart
DBUG[03-19|09:32:05] Initializing a BTRFS driver. 
DBUG[03-19|09:32:05] Mounting BTRFS storage volume for container "mycontainer" on storage pool "default". 
DBUG[03-19|09:32:05] Mounting BTRFS storage pool "default". 
DBUG[03-19|09:32:05] Mounted BTRFS storage volume for container "mycontainer" on storage pool "default". 
DBUG[03-19|09:32:05] Scheduler: container mycontainer started: re-balancing 
DBUG[03-19|09:32:05] 
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {}
	} 
DBUG[03-19|09:32:06] handling                                 ip=@ method=GET url=/1.0
DBUG[03-19|09:32:06] Initializing a BTRFS driver. 
DBUG[03-19|09:32:06] 
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"config": {
				"core.https_address": ":8443",
				"core.trust_password": true
			},
			"api_extensions": [
				"storage_zfs_remove_snapshots",
				"container_host_shutdown_timeout",
				"container_syscall_filtering",
				"auth_pki",
				"container_last_used_at",
				"etag",
				"patch",
				"usb_devices",
				"https_allowed_credentials",
				"image_compression_algorithm",
				"directory_manipulation",
				"container_cpu_time",
				"storage_zfs_use_refquota",
				"storage_lvm_mount_options",
				"network",
				"profile_usedby",
				"container_push",
				"container_exec_recording",
				"certificate_update",
				"container_exec_signal_handling",
				"gpu_devices",
				"container_image_properties",
				"migration_progress",
				"id_map",
				"network_firewall_filtering",
				"network_routes",
				"storage",
				"file_delete",
				"file_append",
				"network_dhcp_expiry",
				"storage_lvm_vg_rename",
				"storage_lvm_thinpool_rename",
				"network_vlan",
				"image_create_aliases"
			],
			"api_status": "stable",
			"api_version": "1.0",
			"auth": "trusted",
			"public": false,
			"environment": {
				"addresses": [
					":8443"
				],
				"architectures": [
					"x86_64",
					"i686"
				],
				"certificate": "-----BEGIN CERTIFICATE-----\nMIIGgjCCBGqgAwIBAgIRAP900K6jIf1qTWYwDrq1lYYwDQYJKoZIhvcNAQELBQAw\nPTEcMBoGA1UEChMTbGludXhjb250YWluZXJzLm9yZzEdMBsGA1UEAwwUcm9vdEBn\ncXVlbnRpbi1zc2QtdDEwHhcNMTYxMDE4MDIzOTM1WhcNMjYxMDE2MDIzOTM1WjA9\nMRwwGgYDVQQKExNsaW51eGNvbnRhaW5lcnMub3JnMR0wGwYDVQQDDBRyb290QGdx\ndWVudGluLXNzZC10MTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBAJb8\nHa2I1Fo1sWKZwXon4OpK/ckDm7HwRtH18+i1yuiddVt/8rhtGkeAMAJhY4rxZb1w\nJoNu2G42JWLQGhtbDOmnbfvNj9DsB+vhpROPJK90K8SFkN4r9kzfW3UIa9LfOg9S\nM5frjdbCEFnSV7rUTm6abMHf2LC6iiVkbsl0psg2Ke7ENlLYS0g8zn7CeNSnYCzC\nYwTj6t0k4uvLAG1YocYVoY1A3z9w2mp2oZEnXgLByB7MmO2JtchXuvH1TDbq5dAH\nH73OOFZxVo5OyBIua+2h3ZPI2bLLqwFehAkKbD8Fd048vwQpBNGARdvcP+nbFqBy\nu7d7w+W5pcR+VvOxC0mphHNIOZ8NeFc0PoIJnNTkF5Nx7i/3omeO7VT53rRc9T6V\n0QcDtM2moXSjwgut7kM/FI3Qh6X2pkRy9TwSOnjOUTdozBxoBGV50tZci1h1BPsB\n1xN9S1PSlDwAUA8YgZN8a8d//xkpsySrLeaTJ8xMlI8GYSt+IEdFuukLMaoYFUhk\nfBpkBRePV611UU2mMW0/bso6bxRSf/dvuEs1gxmMHdKmC9d890OqNtYuwhX2loDV\nodX6KTMYmatDDzlR33xQ29IeXSfsyRlMutSY27MqEgt/u9Xhjj4w8JqysZaRRZYb\n+oi+nCVedFbYRlVpa45eaTzAvzrujB+d9bGNMyBJAgMBAAGjggF7MIIBdzAOBgNV\nHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDAYDVR0TAQH/BAIwADCC\nAUAGA1UdEQSCATcwggEzgg9ncXVlbnRpbi1zc2QtdDGCDjE3Mi4xNi4wLjIwLzEy\nghxmZTgwOjo3NGMyOjVlYjk6M2RmZDo3NGU0LzY0ghxmZTgwOjpiYzAyOjRhZmY6\nZmUwZTo0Y2ViLzY0ghxmZTgwOjpiYzAyOjRhZmY6ZmUwZTo0Y2ViLzY0ghxmZTgw\nOjpiYzAyOjRhZmY6ZmUwZTo0Y2ViLzY0gg4xNzIuMTYuMTEuMS8yNIIcZmU4MDo6\nYmMwMjo0YWZmOmZlMGU6NGNlYi82NIIOMTcyLjE2LjEyLjEvMjSCHGZlODA6OmJj\nMDI6NGFmZjpmZTBlOjRjZWIvNjSCDjE3Mi4xNi4xMC4xLzI0ghxmZTgwOjpmMDkw\nOjM4ZmY6ZmUzNTo3N2RhLzY0gg4xOTIuMTY4LjAuMS8yMDANBgkqhkiG9w0BAQsF\nAAOCAgEAC43ulECKjZxCAcqjfRkHfC/8CTZbp6fLwEsZe/jyWvsYGwoapm6v7qsl\nb8yrwB0+7VqDRHH2aIGG5OJyB2o6FuPVMZnfV+2XEhmeKbP0uybupT/cjy9HLiRa\nKXuWWMDSKuDnwHQx6iw0KiILrZdM2ooc4tw7x+Ed6GUOnQczXrnxXCr7O9WI4cKt\nY1LeY+dv3MPkdrB8dmLf/olPLkJC9hhaYGOWTIQfU2Y96AQIB64/y8duMGcXcStS\n3x8izZx/agDVC3fYr4DrT7+/tqDr1F06SsuP8CZbwDbO51yJob9gJb7SziB/eQqK\n+Kn2d39c+3ohNyTEO1v7wwYL5NbPS3jaKtt2UBiXjVNPCMJoZlJgA6pN8ZcGy5sX\nCcFq9OM0T2+hpq3AIIojodmxTcOGfDcX1IA+rq6pwyCMJahXUwk84FGEmW9LYR6T\nKI5AToUB6MW98ccfmJtdBWzZf9YDmsWfkjxzjKhc3n89wFhBjZPDEHl9772lmICR\nfFeQTezhVwcS5gH50lH162ffprYCICtI3Xwh0a0y9KFXyspabQ5bRzozBHpjRkf0\nHnLsXvkWauRuQtska9kXkljUVl/ZF+h1kO2PRxOor1+VnxBm1XWBEF+0BoRaVQe2\nfgWcrzbDBCtCAY2TVQ18o6vPi+hsK3WjMRHzvKhd3iWGXb4w/hA=\n-----END CERTIFICATE-----\n",
				"certificate_fingerprint": "",
				"driver": "lxc",
				"driver_version": "2.0.7",
				"kernel": "Linux",
				"kernel_architecture": "x86_64",
				"kernel_version": "4.4.0-66-generic",
				"server": "lxd",
				"server_pid": 2681,
				"server_version": "2.11",
				"storage": "btrfs",
				"storage_version": "4.4"
			}
		}
	} 
DBUG[03-19|09:32:06] handling                                 ip=@ method=GET url="/internal/containers/14/onstop?target=stop"
EROR[03-19|09:32:06] stop hook failed                         container=mycontainer err="Container is already running a start operation"
DBUG[03-19|09:32:06] 
	{
		"error": "Container is already running a start operation",
		"error_code": 500,
		"type": "error"
	} 
DBUG[03-19|09:32:06] Scheduler: network: vethQP21PK has been added: updating network priorities 
EROR[03-19|09:32:11] Failed starting container                action=start created=2016-11-10T01:25:00+0000 ephemeral=false name=mycontainer stateful=false used=2017-03-19T08:31:21+0000
DBUG[03-19|09:32:11] Failure for task operation: ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7: Failed to run: /usr/bin/lxd forkstart mycontainer /var/lib/lxd/containers /var/log/lxd/mycontainer/lxc.conf:  
DBUG[03-19|09:32:11] 
	{
		"type": "sync",
		"status": "Success",
		"status_code": 200,
		"operation": "",
		"error_code": 0,
		"error": "",
		"metadata": {
			"id": "ed68d2d1-f3cb-4b0d-a901-c4f6a95ef3c7",
			"class": "task",
			"created_at": "2017-03-19T09:32:05.194592662+01:00",
			"updated_at": "2017-03-19T09:32:05.194592662+01:00",
			"status": "Failure",
			"status_code": 400,
			"resources": {
				"containers": [
					"/1.0/containers/mycontainer"
				]
			},
			"metadata": null,
			"may_cancel": false,
			"err": "Failed to run: /usr/bin/lxd forkstart mycontainer /var/lib/lxd/containers /var/log/lxd/mycontainer/lxc.conf: "
		}
	} 
^CINFO[03-19|09:32:18] Received 'interrupt signal', exiting. 
INFO[03-19|09:32:18] Stopping REST API handler: 
INFO[03-19|09:32:18]  - closing socket                        socket=0.0.0.0:8443
INFO[03-19|09:32:18]  - closing socket                        socket=/var/lib/lxd/unix.socket
INFO[03-19|09:32:18] Stopping /dev/lxd handler 
INFO[03-19|09:32:18] Stopped /dev/lxd handler 
INFO[03-19|09:32:18] Unmounting temporary filesystems 
INFO[03-19|09:32:18] Done unmounting temporary filesystems 
INFO[03-19|09:32:18] Closing the database 
INFO[03-19|09:32:18] Saving simplestreams cache 
INFO[03-19|09:32:18] Saved simplestreams cache 

lxc info --show-log mycontainer

Name: mycontainer
Remote: unix:/var/lib/lxd/unix.socket
Architecture: x86_64
Created: 2016/11/10 01:25 UTC
Status: Stopped
Type: persistent
Profiles: front

Log:

            lxc 20170319083514.857 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:309 - Going to wait for pid 8274.
            lxc 20170319083514.857 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:328 - Trying to sync with child process.
            lxc 20170319083514.858 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 3.
            lxc 20170319083514.858 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 4.
            lxc 20170319083514.858 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 7.
            lxc 20170319083514.858 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:366 - Using pipe file descriptor 6 for monitord.
            lxc 20170319083514.862 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:343 - Sucessfully synced with child process.
            lxc 20170319083514.863 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:312 - Finished waiting on pid 8274.
            lxc 20170319083514.863 INFO     lxc_container - lxccontainer.c:do_lxcapi_start:804 - Attempting to set proc title to [lxc monitor] /var/lib/lxd/containers mycontainer
            lxc 20170319083514.864 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 3.
            lxc 20170319083514.864 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 7.
            lxc 20170319083514.864 INFO     lxc_lsm - lsm/lsm.c:lsm_init:48 - LSM security driver AppArmor
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .reject_force_umount  # comment this to allow umount -f;  not recommended.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:567 - Adding native rule for reject_force_umount action 0.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:do_resolve_add_rule:251 - Setting Seccomp rule to reject force umounts.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:570 - Adding compat rule for reject_force_umount action 0.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:do_resolve_add_rule:251 - Setting Seccomp rule to reject force umounts.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .[all].
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .kexec_load errno 38.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:567 - Adding native rule for kexec_load action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:570 - Adding compat rule for kexec_load action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .open_by_handle_at errno 38.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:567 - Adding native rule for open_by_handle_at action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:570 - Adding compat rule for open_by_handle_at action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .init_module errno 38.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:567 - Adding native rule for init_module action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:570 - Adding compat rule for init_module action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .finit_module errno 38.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:567 - Adding native rule for finit_module action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:570 - Adding compat rule for finit_module action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:402 - processing: .delete_module errno 38.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:567 - Adding native rule for delete_module action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:570 - Adding compat rule for delete_module action 327718.
            lxc 20170319083514.864 INFO     lxc_seccomp - seccomp.c:parse_config_v2:580 - Merging in the compat Seccomp ctx into the main one.
            lxc 20170319083514.864 INFO     lxc_conf - conf.c:run_script_argv:424 - Executing script "/usr/bin/lxd callhook /var/lib/lxd 14 start" for container "mycontainer", config section "lxc".
            lxc 20170319083514.864 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:309 - Going to wait for pid 8278.
            lxc 20170319083514.865 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:328 - Trying to sync with child process.
            lxc 20170319083514.865 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 3.
            lxc 20170319083514.865 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 4.
            lxc 20170319083514.865 INFO     lxc_start - start.c:lxc_check_inherited:235 - Closed inherited fd: 7.
            lxc 20170319083514.865 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:366 - Using pipe file descriptor 6 for monitord.
            lxc 20170319083514.872 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:343 - Sucessfully synced with child process.
            lxc 20170319083514.873 DEBUG    lxc_monitor - monitor.c:lxc_monitord_spawn:312 - Finished waiting on pid 8278.
            lxc 20170319083514.873 INFO     lxc_monitor - monitor.c:lxc_monitor_sock_name:185 - Using monitor socket name "lxc/d78a9d7e97b4b375//var/lib/lxd/containers".
            lxc 20170319083514.913 DEBUG    lxc_start - start.c:setup_signal_fd:273 - Set SIGCHLD handler with file descriptor: 4.
            lxc 20170319083514.913 DEBUG    console - console.c:lxc_console_peer_default:468 - no console peer
            lxc 20170319083514.913 INFO     lxc_start - start.c:lxc_init:475 - Container "mycontainer" is initialized.
            lxc 20170319083514.914 DEBUG    lxc_start - start.c:__lxc_start:1317 - Not dropping CAP_SYS_BOOT or watching utmp.
            lxc 20170319083514.917 INFO     lxc_conf - conf.c:instantiate_veth:2647 - Retrieved mtu 1500 from lxdbr0
            lxc 20170319083514.918 INFO     lxc_conf - conf.c:instantiate_veth:2672 - Attached 'vethI6JJMR': to the bridge 'lxdbr0': 
            lxc 20170319083514.918 DEBUG    lxc_conf - conf.c:instantiate_veth:2689 - instantiated veth 'vethI6JJMR/vethFQETIJ', index is '87'
            lxc 20170319083514.926 INFO     lxc_conf - conf.c:instantiate_veth:2647 - Retrieved mtu 1500 from lxdbr1
            lxc 20170319083514.927 INFO     lxc_conf - conf.c:instantiate_veth:2672 - Attached 'vethCCUPLJ': to the bridge 'lxdbr1': 
            lxc 20170319083514.928 DEBUG    lxc_conf - conf.c:instantiate_veth:2689 - instantiated veth 'vethCCUPLJ/vethY8NKS8', index is '89'
            lxc 20170319083514.931 INFO     lxc_conf - conf.c:instantiate_veth:2647 - Retrieved mtu 1500 from lxdbr2
            lxc 20170319083514.932 INFO     lxc_conf - conf.c:instantiate_veth:2672 - Attached 'veth8WWKXT': to the bridge 'lxdbr2': 
            lxc 20170319083514.932 DEBUG    lxc_conf - conf.c:instantiate_veth:2689 - instantiated veth 'veth8WWKXT/veth9K0YA8', index is '91'
            lxc 20170319083514.932 INFO     lxc_cgroup - cgroups/cgroup.c:cgroup_init:68 - cgroup driver cgroupfs-ng initing for mycontainer
            lxc 20170319083514.933 DEBUG    lxc_cgfsng - cgroups/cgfsng.c:filter_and_set_cpus:474 - No isolated cpus detected.
            lxc 20170319083514.933 DEBUG    lxc_cgfsng - cgroups/cgfsng.c:handle_cpuset_hierarchy:644 - "cgroup.clone_children" was already set to "1".
            lxc 20170319083514.933 INFO     lxc_start - start.c:lxc_spawn:1154 - Cloned CLONE_NEWUSER.
            lxc 20170319083514.933 INFO     lxc_start - start.c:lxc_spawn:1154 - Cloned CLONE_NEWNS.
            lxc 20170319083514.933 INFO     lxc_start - start.c:lxc_spawn:1154 - Cloned CLONE_NEWPID.
            lxc 20170319083514.933 INFO     lxc_start - start.c:lxc_spawn:1154 - Cloned CLONE_NEWUTS.
            lxc 20170319083514.933 INFO     lxc_start - start.c:lxc_spawn:1154 - Cloned CLONE_NEWIPC.
            lxc 20170319083514.941 INFO     lxc_start - start.c:do_start:745 - Unshared CLONE_NEWNET.
            lxc 20170319083514.977 INFO     lxc_confile - confile.c:config_idmap:1531 - read uid map: type u nsid 0 hostid 362144 range 65536
            lxc 20170319083514.977 INFO     lxc_confile - confile.c:config_idmap:1531 - read uid map: type g nsid 0 hostid 362144 range 65536
            lxc 20170319083515.979 DEBUG    lxc_conf - conf.c:lxc_assign_network:3185 - move 'vethFQETIJ'/'eth0' to '8350': .
            lxc 20170319083515.369 DEBUG    lxc_conf - conf.c:lxc_assign_network:3185 - move 'vethY8NKS8'/'eth1' to '8350': .
            lxc 20170319083515.689 DEBUG    lxc_conf - conf.c:lxc_assign_network:3185 - move 'veth9K0YA8'/'eth2' to '8350': .
            lxc 20170319083515.690 NOTICE   lxc_utils - utils.c:lxc_switch_uid_gid:2064 - Switched to gid 0.
            lxc 20170319083515.690 NOTICE   lxc_utils - utils.c:lxc_switch_uid_gid:2070 - Switched to uid 0.
            lxc 20170319083515.690 NOTICE   lxc_utils - utils.c:lxc_setgroups:2082 - Dropped additional groups.
            lxc 20170319083515.693 ERROR    lxc_conf - conf.c:mount_rootfs:865 - Permission denied - failed to get real path for '/var/lib/lxd/containers/mycontainer/rootfs'
            lxc 20170319083515.694 ERROR    lxc_conf - conf.c:setup_rootfs:1279 - failed to mount rootfs
            lxc 20170319083515.694 ERROR    lxc_conf - conf.c:do_rootfs_setup:3750 - failed to setup rootfs for 'mycontainer'
            lxc 20170319083515.694 ERROR    lxc_conf - conf.c:lxc_setup:3832 - Error setting up rootfs mount after spawn
            lxc 20170319083515.694 ERROR    lxc_start - start.c:do_start:811 - Failed to setup container "mycontainer".
            lxc 20170319083515.694 ERROR    lxc_sync - sync.c:__sync_wait:57 - An error occurred in another process (expected sequence number 3)
            lxc 20170319083515.695 INFO     lxc_conf - conf.c:lxc_delete_network:3005 - Interface "eth0" with index 87 already deleted or existing in different network namespace.
            lxc 20170319083515.124 INFO     lxc_conf - conf.c:lxc_delete_network:3040 - Removed interface "vethI6JJMR" from host.
            lxc 20170319083515.165 INFO     lxc_conf - conf.c:lxc_delete_network:3005 - Interface "eth1" with index 89 already deleted or existing in different network namespace.
            lxc 20170319083515.167 WARN     lxc_conf - conf.c:lxc_delete_network:3038 - Failed to remove "vethCCUPLJ" from host: Invalid argument.
            lxc 20170319083515.167 INFO     lxc_conf - conf.c:lxc_delete_network:3005 - Interface "eth2" with index 91 already deleted or existing in different network namespace.
            lxc 20170319083515.169 WARN     lxc_conf - conf.c:lxc_delete_network:3038 - Failed to remove "veth8WWKXT" from host: Invalid argument.
            lxc 20170319083515.169 ERROR    lxc_start - start.c:__lxc_start:1346 - Failed to spawn container "mycontainer".
            lxc 20170319083515.228 INFO     lxc_conf - conf.c:run_script_argv:424 - Executing script "/usr/share/lxcfs/lxc.reboot.hook" for container "mycontainer", config section "lxc".
            lxc 20170319083515.731 INFO     lxc_conf - conf.c:run_script_argv:424 - Executing script "/usr/bin/lxd callhook /var/lib/lxd 14 stop" for container "mycontainer", config section "lxc".
            lxc 20170319083515.753 ERROR    lxc_conf - conf.c:run_buffer:405 - Script exited with status 1.
            lxc 20170319083515.753 ERROR    lxc_start - start.c:lxc_fini:546 - Failed to run lxc.hook.post-stop for container "mycontainer".
            lxc 20170319083515.753 WARN     lxc_commands - commands.c:lxc_cmd_rsp_recv:172 - Command get_cgroup failed to receive response: Connection reset by peer.
            lxc 20170319083515.753 WARN     lxc_commands - commands.c:lxc_cmd_rsp_recv:172 - Command get_cgroup failed to receive response: Connection reset by peer.
            lxc 20170319083527.392 INFO     lxc_confile - confile.c:config_idmap:1531 - read uid map: type u nsid 0 hostid 362144 range 65536
            lxc 20170319083527.392 INFO     lxc_confile - confile.c:config_idmap:1531 - read uid map: type g nsid 0 hostid 362144 range 65536
            lxc 20170319083527.401 INFO     lxc_confile - confile.c:config_idmap:1531 - read uid map: type u nsid 0 hostid 362144 range 65536
            lxc 20170319083527.401 INFO     lxc_confile - confile.c:config_idmap:1531 - read uid map: type g nsid 0 hostid 362144 range 65536

**Comment by stgraber** [comment]
There's a permission problem with the path leading to: /var/lib/lxd/containers/mycontainer/rootfs

Please post the output of:
```
sudo ls -ld / /var /var/lib /var/lib/lxd /var/lib/lxd/containers /var/lib/lxd/containers/mycontainer /var/lib/lxd/storage-pools /var/lib/lxd/storage-pools/default /var/lib/lxd/storage-pools/default/containers /var/lib/lxd/storage-pools/default/containers/mycontainer /var/lib/lxd/storage-pools/default/containers/mycontainer/rootfs
```

**Comment by gaetanquentin** [comment]
ls -ld / /var /var/lib /var/lib/lxd /var/lib/lxd/containers /var/lib/lxd/containers/services /var/lib/lxd/storage-pools /var/lib/lxd/storage-pools/default /var/lib/lxd/storage-pools/default/containers /var/lib/lxd/storage-pools/default/containers/services /var/lib/lxd/storage-pools/default/containers/services/rootfs
drwxr-xr-x  1 root   root    290 mars   8 04:43 /
drwxr-xr-x  1 root   root    134 févr. 15 09:15 /var
drwxr-xr-x  1 root   root   1642 févr. 15 12:40 /var/lib
drwxr-xr-x  1 root   root    254 mars  19 09:35 /var/lib/lxd
drwx--x--x  1 root   root    358 mars  18 20:17 /var/lib/lxd/containers
lrwxrwxrwx  1 root   root     54 mars   6 21:10 /var/lib/lxd/containers/services -> /var/lib/lxd/storage-pools/default/containers/services
drwx--x--x  1 root   root     14 mars   6 09:07 /var/lib/lxd/storage-pools
drwxr-xr-x  1 root   root     62 mars   6 09:07 /var/lib/lxd/storage-pools/default
drwx------  1 root   root    326 mars  18 20:17 /var/lib/lxd/storage-pools/default/containers
drwxr-xr-x+ 1 362144 362144   78 févr. 15 13:32 /var/lib/lxd/storage-pools/default/containers/services
drwxr-xr-x  1 362144 362144  148 janv. 18 23:43 /var/lib/lxd/storage-pools/default/containers/services/rootfs



subid/subgid:

lxd:362144:65536

**Comment by stgraber** [comment]
sudo chmod +x /var/lib/lxd/storage-pools/default/containers

That should fix it

**Comment by stgraber** [comment]
@brauner any idea what caused this? ^

**Comment by brauner** [comment]
@gaetanquentin, is this an upgraded LXD instance or a freshly installed one?

**Comment by gaetanquentin** [comment]
upgraded.

**Comment by gaetanquentin** [comment]
ok it works with chmod +x /var/lib/lxd/storage-pools/default/containers

**Comment by stgraber** [comment]
Just did a quick clean test upgrade and this seems fine. So I suspect it happened as a result of hopping through some of our buggier releases... Going to close this for now, if we see similar reports we'll add an extra migration step to try and fix those.

**Comment by gaetanquentin** [comment]
FYI It happened again with lxd 2.13 i think  (upgraded version) on ubuntu 17.04.

sudo chmod +x /var/lib/lxd/storage-pools/default/containers fixed it again.
